modules/common/src/main/MarkdownRender.scala

```diff
@@ -204,13 +206,22 @@ object MarkdownRender:
     private val gameRegex =
       s"""^(?:https?://)?${expander.domain}/(?:embed/)?(?:game/)?(\\w{8})(?:(?:/(white|black))|\\w{4}|)(?:#(\\d+))?$$""".r
 
+    private val gameInSiteHeader = 
```

Comment:
We can avoid additional regex here with access to the flexmark renderer's Link which I believe has startOffset as a public method (via Node ancestry).  Maybe a solution that just subtracts length of "[Site \"" for an exact string match would be better.

Reply:
Thank you @schlawg ! I will follow this way to not add another regex.